examples/rgw-cache/nginx.conf

```
@@ -0,0 +1,40 @@
+
+user  nginx;
+#Process per core
+worker_processes  auto;
+pid        /var/run/nginx.pid;
+
+events {
+#Number of connections per worker
+    worker_connections  1024;
+}
+
+
+http {
```

```suggestion
http {
    types_hash_max_size 4096;
```
had to increase to 4096 to resolve the warning